modules/local/report.nf

```diff
@@ -245,18 +245,21 @@ def create_action_call(sample_data, species_tag){
                 // nothing to do here
                 if(params.metagenomic_run){
                     final_message = "No QC Summary is provided for metagenomic samples."
+                    qc_summary = "No quality control criteria is applied for metagenomic samples."
```

Comment:
Could you add test cases for this? E.g., in this case adding a case to set `params.metagenomic_run` and make sure this message is returned.

Do you have any small data (or ways to update small datasets) so it forces detection of metagenomic samples?

Reply:
I can make something small! I may be able to spoof some inputs.

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/103/commits/3b07a9be11a0aff6928d8269e4eb17a08e63d496